features/step_definitions/user_steps.rb

```diff
@@ -107,6 +107,10 @@
   user.profile.save!
 end
 
+Given /^I have a limited post with text "([^\"]*)" in the aspect "([^"]*)"$/ do |text, aspect_name|
+  post = @me.post :status_message, text: text, to: @me.aspects.where(name: aspect_name).first.id
```

Comment:
Well yeah, this is weird, I forgot to `post.save` here but the test is green anyway oO, [this line](https://github.com/diaspora/diaspora/pull/7265/files#diff-b0f17d137a09c6491e446c550000d479R66) should fail...